### DISCUSSION TITLE
Playlist duration

### DISCUSSION THREAD

**Discussion by audio-drome** [rern/rAudio#1186] [Feature Requests]
Hi,

I think it would be useful to have shown the duration of the playlist in use.
I think it could be added in the playlist view (there's some space in the upper bar).

BTW, thanks for the excellent job!

Zeno

**Comment by klassikmann** [comment]
@audio-drome 
This feature is already present as you propose in the upper bar. On small devices as a smartphone you have to turn the device to landscape mode to see the playlist duration.

**Reply by audio-drome** [reply]
Sorry, I use a smartphone as remote controller but I quite never use it in landscape mode so I didn't notice that the information about duration was already there.

Anyway, having the information also in portrait mode would be top!

Thank you again and bye.
